doc/src/examples.rst

```diff
@@ -0,0 +1,157 @@
+
+1. Create a working directory called ``euler_vortex_2d/``
+
+2. Copy the configuration file
+   ``PyFR/examples/euler_vortex_2d/euler_vortex_2d.ini`` into
+   ``euler_vortex_2d/``
+
+3. Copy the `Gmsh <http:http://geuz.org/gmsh/>`_ file
+   ``PyFR/examples/euler_vortex_2d/euler_vortex_2d.msh`` into
+   ``euler_vortex_2d/``
+
+4. Run pyfr to convert the `Gmsh <http:http://geuz.org/gmsh/>`_
+   mesh file into a PyFR mesh file called ``euler_vortex_2d.pyfrm``::
+
+        pyfr import euler_vortex_2d.msh euler_vortex_2d.pyfrm
+
+5. Run pyfr to partition the PyFR mesh file into two pieces::
+
+        pyfr partition 2 euler_vortex_2d.pyfrm .
+
+6. Run pyfr to solve the Euler equations on the mesh, generating a
+   series of PyFR solution files called ``euler_vortex_2d*.pyfrs``::
+
+        mpiexec -n 2 pyfr run -b cuda -p euler_vortex_2d.pyfrm euler_vortex_2d.ini
+
+7. Run pyfr on the solution file ``euler_vortex_2d-100.0.pyfrs``
+   converting it into an unstructured VTK file called
+   ``euler_vortex_2d-100.0.vtu``. To visualise the results using 
+   subdivision, a control integer is added at the end of the 
+   command::
+
+        pyfr export euler_vortex_2d.pyfrm euler_vortex_2d-100.0.pyfrs euler_vortex_2d-100.0.vtu -d 4
+
+8. Visualise the unstructured VTK file in `Paraview
+   <http://www.paraview.org/>`_
+
+.. figure:: ../fig/euler_vortex_2d/euler_vortex_2d.png
+   :width: 450px
+   :figwidth: 450px
+   :alt: euler vortex
+   :align: center
+
+   Colour map of density distribution at 100 time units.
+
+Compressible Navier--Stokes Equations
+=====================================
+
+2D Couette Flow
+---------------
+
+Proceed with the following steps to run a serial 2D Couette flow
+simulation on a mixed unstructured mesh:
+
+1. Create a working directory called ``couette_flow_2d/``
+
+2. Copy the configuration file
+   ``PyFR/examples/couette_flow_2d/couette_flow_2d.ini`` into
+   ``couette_flow_2d/``
+
+3. Copy the `Gmsh <http:http://geuz.org/gmsh/>`_ mesh file
+   ``PyFR/examples/couette_flow_2d/couette_flow_2d.msh`` into
+   ``couette_flow_2d/``
+
+4. Run pyfr to covert the `Gmsh <http:http://geuz.org/gmsh/>`_
+   mesh file into a PyFR mesh file called ``couette_flow_2d.pyfrm``::
+
+        pyfr import couette_flow_2d.msh couette_flow_2d.pyfrm
+
+5. Run pyfr to solve the Navier-Stokes equations on the mesh,
+   generating a series of PyFR solution files called
+   ``couette_flow_2d-*.pyfrs``::
+
+        pyfr run -b cuda -p couette_flow_2d.pyfrm couette_flow_2d.ini
+
+6. Run pyfr on the solution file ``couette_flow_2d-040.pyfrs``
+   converting it into an unstructured VTK file called
+   ``couette_flow_2d-040.vtu``. Note that in order to visualise the
+   high-order data, each high-order element is sub-divided into smaller
+   linear elements. The level of sub-division is controlled by the
+   integer at the end of the command::
+
+        pyfr export couette_flow_2d.pyfrm couette_flow_2d-040.pyfrs couette_flow_2d-040.vtu -d 4
```

Comment:
Remove subdivision.

Reply:
Is there an argument for leaving the subdivision in so that the .vtu files will work with older version of Paraview that don't support high-order cells?

Reply:
On balance probably not.  That one argument gives rise to two additional explanatory sentences; one of which is no longer true (we can visualise high-order data directly and, indeed, it is the recommended solution by default).

Reply:
OK, I agree with removing the -d 4 (along with the explanatory sentences).

Reply:
Probably worth mentioning here to increase the 'Nonlinear Subdivision level' in Paraview otherwise I think the rendering will not show the high-order solution by default. We also may need to update the pictures for each example.